mysql-router/dockerdir/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
I prefer having general docker image (maybe even official) and all specific k8s things should be in operator repo or it should work for everyone by default. So, please create universal 'entrypoint' or general one in this repo and for operator needs in our operator repo.

Reply:
how about we do the same thing as orchestrator entrypoint, we can check `$KUBERNETES_SERVICE_HOST` and do k8s logic else just run `exec $@`

Reply:
It is ok for me.

Reply:
https://github.com/percona/percona-docker/pull/628/commits/b77d4647c85aa65217781392fccdbc65e09c7ce5